CHANGELOG.md

```diff
@@ -12,7 +12,9 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 	https://changelog.md/
 -->
 
-## v4.0.1 (WIP)
+## v4.1.0 (WIP)
+
+- Add endpoint PUT `PUT /provider`. (#28)
```

Comment:
nit: Double wording of "PUT". Also, not a requirement, but would be nice to say a little what the new endpoint does.

```suggestion
- Added endpoint `PUT /provider` as an idempotent way of creating a
  provider. (#28)
```